From 64edb1c2ec31121b8b9cc5b4a75eab384fed2f2c Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@turris.com>
Date: Mon, 22 Apr 2024 21:06:39 +0200
Subject: [PATCH] packages: iftop: bump to newer version

Because iftop package was not reinstalled automatically during system upgrade
and showed negative numbers of network traffic, easy fix is to bump it to the
latest version
---
 net/iftop/Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/net/iftop/Makefile b/net/iftop/Makefile
index 771afa996..68abad2da 100644
--- a/net/iftop/Makefile
+++ b/net/iftop/Makefile
@@ -1,13 +1,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=iftop
-PKG_RELEASE:=2
+PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://code.blinkace.com/pdw/iftop.git
-PKG_SOURCE_DATE:=2018-10-03
-PKG_SOURCE_VERSION:=77901c8c53e01359d83b8090aacfe62214658183
-PKG_MIRROR_HASH:=219231541a437f5aecd497796be0202d337e13f141359a93595bf2cd8c5c5544
+PKG_SOURCE_DATE:=2020-04-11
+PKG_SOURCE_VERSION:=8d76ef06d225c4d72db544b54ea1ef1a0e077967
+PKG_MIRROR_HASH:=d444793fd0826423b3b2740dfa7113ec2e9a67cb49e33bef9cdd78657dc4502b,
 PKG_MAINTAINER:=Paul Spooren <mail@aparcar.org>
 PKG_LICENSE:=GPL-2.0
 
-- 
2.44.0

